packages/rum-core/src/domain/rumEventsCollection/action/listenActionEvents.ts

```diff
@@ -109,14 +83,11 @@ function isSelectionEmpty(): boolean {
   return !selection || selection.isCollapsed
 }
 
-function isValidMouseEvent(event: MouseEvent): event is MouseEventOnElement {
+function isValidPointerEvent(event: PointerEvent): event is MouseEventOnElement {
   return (
     event.target instanceof Element &&
     // Only consider 'primary' pointer events for now. Multi-touch support could be implemented in
     // the future.
-    // On Chrome, click events are PointerEvent with `isPrimary = false`, but we should still
-    // consider them valid. This could be removed when we enable the `click-action-on-pointerup`
-    // flag, since we won't rely on click events anymore.
-    (event.type === 'click' || (event as PointerEvent).isPrimary !== false)
+    event.isPrimary !== false
   )
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏do we need to wrap the condition by parentheses?
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏could we directly use `event.isPrimary` or do we need to explicitly check `event.isPrimary !== false?`

Reply:
> **🥜 nitpick:** ‏do we need to wrap the condition by parentheses?

There are parentheses because the condition is on multiple lines and prettier format it like that. I could move the comment above the `return` statement, but since it's only related to `isPrimary` I think it would be better to keep it like that

> 
> **❓ question:** ‏could we directly use `event.isPrimary` or do we need to explicitly check `event.isPrimary !== false?`

we could, but then when emulating clicks (ex: in tests in the SDK or outside of it, in Synthetics test for the demo org...) we would need to always set `isPrimary: true` on those events, which is not a huge deal but still something more to take into account

Reply:
ok!